package.json

```diff
@@ -51,17 +51,17 @@
   },
   "license": "MIT",
   "scripts": {
+    "fabric": "node ./scripts/index.mjs",
```

Comment:
can we call this CLI or something else that is not fabric? sounds confusing to me.

Reply:
done